src/Stack/GhcPkg.hs

```diff
@@ -11,6 +11,7 @@ module Stack.GhcPkg
   ,findGhcPkgField
   ,createDatabase
   ,unregisterGhcPkgId
```

Comment:
Maybe change this to

```suggestion
  ,unregisterSingleGhcPkgId
```

(and rename the function at line 151, where I can't comment) as it is clearer than the only difference being the plural suffix 's' and is consistent with `unregisterSinglePackage` vs `unregisterPackages`.

(note: github still haven't fixed the bug with indentation in their suggestions.)